File: beginner/pagina11/2850/2850.java
import java.util.Scanner;
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner in = new Scanner(system.in);

        while(in.hasNextLine()){
            String entrada = in.nextLine();
            if(entrada.equals("esquerda")){System.out.println( "ingles"); }
            if(entrada.equals("direita")){ System.out.println("frances"); }
            if(entrada.equals("nenhuma")){ System.out.println("portugues"); }
            if(entrada.equals("as duas")){ System.out.println("caiu"); }
        }

        in.close();
    }
}

File: beginner/pagina11/2850/2850.py
while True:
    try:
        entrada = input()
        if entrada == "esquerda":
            print("ingles")
        if entrada == "direita":
            print("frances")
        if entrada == "nenhuma":
            print("portugues")
        if entrada == "as duas":
            print("caiu")

    except EOFError:
        break